subprojects/model-core/src/integTest/groovy/org/gradle/api/provider/CredentialsProviderIntegrationTest.groovy

```diff
@@ -237,9 +237,10 @@ class CredentialsProviderIntegrationTest extends AbstractIntegrationSpec {
         }
 
         where:
-        credentialsType       | errorMessage
-        'AwsCredentials'      | "The following Gradle properties are missing for 'test' credentials:\n  - testAccessKey\n  - testSecretKey"
-        'PasswordCredentials' | "The following Gradle properties are missing for 'test' credentials:\n  - testUsername\n  - testPassword"
+        credentialsType         | errorMessage
+        'AwsCredentials'        | "The following Gradle properties are missing for 'test' credentials:\n  - testAccessKey\n  - testSecretKey"
+        'PasswordCredentials'   | "The following Gradle properties are missing for 'test' credentials:\n  - testUsername\n  - testPassword"
+        'HttpHeaderCredentials' | "The following Gradle properties are missing for 'test' credentials:\n  - testName\n  - testValue"
```

Comment:
Following the name changes, the expected error message needs to be modified as well

Reply:
This should be fixed now, apologies for missing this